errors/errors.go

```diff
@@ -228,3 +233,76 @@ func Is(a, b error) bool {
 	}
 	return ac.ABCICode() == bc.ABCICode()
 }
+
+//---- Stacktrace formatting -----
+
+func matchesFile(f errors.Frame, substrs ...string) bool {
+	file, _ := fileLine(f)
+	for _, sub := range substrs {
+		if strings.Contains(file, sub) {
+			return true
+		}
+	}
+	return false
+}
+
+func fileLine(f errors.Frame) (string, int) {
+	pc := uintptr(f) - 1
```

Comment:
Magic! Please add a short comment here so that it's clear for everybody what happens here.

Reply:
I don't really know.. But I van link to the source code I lifted this trick from

Reply:
I think I do understand what happens threre, but it might not be clear to others who try to read this:) 

Yeah, let’s provide a link and I will be happy extend the doc later, once I have played around with it a bit.